mobile/lib/shared/views/backup_controller_page.dart

```diff
@@ -17,6 +22,32 @@ class BackupControllerPage extends HookConsumerWidget {
     BackUpState _backupState = ref.watch(backupProvider);
     AuthenticationState _authenticationState = ref.watch(authenticationProvider);
 
+    // Select default album if not set
+    bool hasBackupInfo = Hive.box<HiveSavedBackupInfo>(hiveBackupInfoBox).containsKey(savedBackupInfoKey);
+    if (!hasBackupInfo) {
+      Hive.box<HiveSavedBackupInfo>(hiveBackupInfoBox).put(
+          savedBackupInfoKey,
+          HiveSavedBackupInfo(
+            assetEntityId: "isAll",
```

Comment:
This would cause a problem on the first launch on iOS since it set an invalid `albumId` so we can remove line 26-34